src/MOIWrapper.jl

```diff
@@ -0,0 +1,561 @@
+                rows::Vector{Int}, cols::Vector{Int}, coefs::Vector{Float64})
+                
+    elements = Float64[]
+    columns = Int32[]
+    
+    number_in_row = 0
+    first = rows[row]
+    last = (row==length(rows)) ? length(cols) : rows[row+1]-1
+    for i in first:last
+        number_in_row += 1
+        push!(elements, coefs[i])
+        push!(columns, cols[i] - 1)
+    end  
+    
+    add_row(instance.inner, Cint(number_in_row), columns, elements, lower, upper)
+end
+
+"""
+    add_linear_constraints!(instance, A::CSRMatrix{Float64}, sense::Vector{Cchar}, 
+        rhs::Vector{Float64})::Void
+
+Adds linear constraints of the form `Ax (sense) rhs` to the model `m`.
+`sense` and `rhs` contain one element for each row in `A`.
+"""
+function LQOI.add_linear_constraints!(instance::ClpOptimizer, A::LQOI.CSRMatrix{Float64}, 
+        sense::Vector{Cchar}, rhs::Vector{Float64})::Void
+                        
+    rows = A.row_pointers
+    cols = A.columns
+    coefs = A.coefficients
+           
+    num_rows = length(rhs)
+    for row in 1:num_rows        
+        if (rhs[row] > 1e20)
+            error("rhs must always be less than 1e20")
+        elseif (rhs[row] < -1e20)
+            error("rhs must always be greater than -1e20")
+        end
+        
+        lower = -Inf
+        upper = Inf                        
+        if sense[row] == Cchar('L')
+            upper = rhs[row]
+        elseif sense[row] == Cchar('G')
+            lower = rhs[row]
+        elseif sense[row] == Cchar('E')
+            upper = lower = rhs[row]
+        else
+            error("sense must be Cchar(x) where x is in ['L','G',E']")
+        end
+        
+        append_row(row, lower, upper, instance, rows, cols, coefs)
+    end
+end
+
+"""
+    add_ranged_constraints!(instance, A::CSRMatrix{Float64}, lb::Vector{Float64}, 
+        ub::Vector{Float64})::Void
+
+Adds linear constraints of the form `Ax (sense) rhs` to the model `m`.
+`sense` and `rhs` contain one element for each row in `A`.
+"""
+function LQOI.add_ranged_constraints!(instance::Clp.ClpOptimizer, 
+        A::LinQuadOptInterface.CSRMatrix{Float64}, 
+        lb::Vector{Float64}, ub::Vector{Float64})
+
+    rows = A.row_pointers
+    cols = A.columns
+    coefs = A.coefficients
+           
+    num_rows = length(lb)
+    for row in 1:num_rows
+        lower = lb[row]
+        upper = ub[row]
+    
+        append_row(row, lower, upper, instance, rows, cols, coefs)                
+    end
+end
+
+
+"""
+    get_rhs(instance, row::Int)::Float64
+
+Get the right-hand side of the linear constraint in the 1-indexed row `row` in
+the model `m`.
+"""
+function LQOI.get_rhs(instance::ClpOptimizer, row::Int)::Float64
+    lower = replace_inf(get_row_lower(instance.inner))
+    upper = replace_inf(get_row_upper(instance.inner))
+    lb = lower[row]
+    ub = upper[row]
+    
+    if lb > -Inf
+        return lb
+    elseif ub < Inf
+        return ub
+    else
```

Comment:
What about interval constraints? You need to implement [get_range](https://github.com/JuliaOpt/LinQuadOptInterface.jl/blob/d6957f357227e692c0e15bd6c64e9d0a37b7b1f1/src/solver_interface.jl#L198)

Reply:
And also [modify_ranged_constraints!](https://github.com/JuliaOpt/LinQuadOptInterface.jl/blob/d6957f357227e692c0e15bd6c64e9d0a37b7b1f1/src/solver_interface.jl#L182)